README.md

```diff
@@ -336,6 +498,51 @@ return [
 ];
 ```
 
+#### Re-generating the routes file when your app routes change
+
+Watch your app's route files, and whenever they're saved, re-run `php artisan ziggy:generate` automatically. The example is a Laravel Mix plugin but you could do it without Mix too. Huge thanks to [Nuno Rodrigues](https://github.com/nacr) for [the idea and a sample implementation](https://github.com/tighten/ziggy/issues/321#issuecomment-689150082)!
```

Comment:
I think here you meant to write something like "If you're exporting your Ziggy routes as a file, you may want to watch"